app/services/halls/delete.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Halls
+  class Delete
+    def initialize(hall_id)
+      @hall_id = hall_id
+    end
+
+    def call
+      hall = Hall.find(hall_id)
+      hall.destroy
+    end
```

Comment:
```suggestion
    def call(hall_id)
      hall = Hall.find(hall_id)
      hall.destroy
    end
```

In such a short classes you can pass a `hall_id` to call method and remove all other things in this class.
You can think it over and make your decision.